core/src/main/java/com/scalar/db/sql/Value.java

```diff
@@ -30,8 +31,14 @@ public boolean equals(Object o) {
     if (!(o instanceof Value)) {
       return false;
     }
-    Value value1 = (Value) o;
-    return type == value1.type && Objects.equals(value, value1.value);
+    Value that = (Value) o;
+    if (type != that.type) {
+      return false;
+    }
+    if (type == Type.BLOB_BYTES) {
+      return Arrays.equals((byte[]) value, (byte[]) that.value);
+    }
```

Comment:
When the `type` is `BLOB_BYTES`, the `value` is a `byte[]` type, so we need to use `Arrays.equals()`.